Source/WebKit/NetworkProcess/NetworkProcessCreationParameters.serialization.in

```diff
@@ -56,6 +56,7 @@ headers: "NetworkProcessCreationParameters.h" "AuxiliaryProcessCreationParameter
 #if ENABLE(BUILT_IN_NOTIFICATIONS)
     bool builtInNotificationsEnabled
 #endif
+    bool usesSingleWebProcess
```

Comment:
For Cocoa ports, the network process is a singleton. However, the `usesSingleWebProcess` is per ProcessPool in the UIProcess. As a result, having a global `usesSingleWebProcess` for the whole network process isn't quite right.

Once way to address this would be to move `usesSingleWebProcess` to `NetworkProcessPreferencesForWebProcess` instead. I'm trying to think if there is an easier way but at least having the flag on `NetworkProcessPreferencesForWebProcess` would be correct.